### ISSUE TITLE
The camera have coordinate system in chunks

### ISSUE THREAD

**Issue by izissise** [izissise/PFA#19]
Each time the camera move to the next chunk coordinate get 1 unit, how can I get a coordinate system where a title is the unit ?

**Comment by Tastyep** [comment]
you have the player's chunk id and the player's cordinate in the chunk (1,1),
Do something like (chunkId + pos) * {Chunk::width, Chunk::height}

**Comment by izissise** [comment]
Yep, but I don't like depending on the chunk in the draw stage, I'll find a way, thank you
